src/components/Main/Loader/index.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import ClipLoader from "react-spinners/ClipLoader";
+
+const Loader = () => {
+  return (
+    <Div>
```

Comment:
Do not name custom styled or simple components as default that already exists in HTML

```suggestion
    <CustomDivStyled>
```